File: frontend/src/constants/Representative.ts
import { Dayjs } from "dayjs";

export type Representative = {
  
  id: number;
  firstName: string;
  secondName: string;
  pin: string;
  companyId: number;
  hasAccess: boolean;
  typeId: number;
  lastName: string;
};


export type RepresentativeCreateModel = {
  
  id: number;
  firstName: string;
  secondName: string;
  pin: string;
  companyId: number;
  hasAccess: boolean;
  typeId: number;
  lastName: string;
};

File: backend/Domain/Entities/Representative.cs

namespace Domain.Entities
{
    public class Representative : BaseLogDomain
    {
        public int Id { get; set; }
		public string FirstName { get; set; }
		public string SecondName { get; set; }
		public string Pin { get; set; }
		public int CompanyId { get; set; }
		public bool? HasAccess { get; set; }
		public int TypeId { get; set; }
		public string LastName { get; set; }
		
    }
}